app/controllers/comments_controller.rb

```diff
@@ -21,6 +21,6 @@ def destroy
   private
 
   def comment_params
-    params.require(:comment).permit(:commenter, :body, :status)
+    params.require(:comment).permit(:body, :status).merge(commenter: helpers.current_user.username)
```

Comment:
@JuanMaGiraldo this line will raise an error, please correct it

Reply:
Fixed! thanks.